packages/components/src/text-area/src/TextArea.tsx

```diff
@@ -61,6 +61,36 @@ const pxToInt = (value?: string) => {
     return !isNil(value) ? parseInt(value.replace("px", ""), 10) : 0;
 };
 
+function useCalculateLineHeight(inputRef: any) {
```

Comment:
Pass only the inputRef.current, and type it with 

`inputRef.current as HTMLTextAreaElement`